### PR TITLE
feat: add tags

### PR DESCRIPTION
**Pull request by aniketsingh13**
chrome-capture-2022-4-5

### CODE REVIEW

File: note-app/src/Reducer/setnoteReducer.js [hunk 1]
```diff
@@ -12,11 +12,16 @@ const setnoteReducer = (state,action) => {
           return {...state,noteColor:action.payload}
           case "COLORPALLET":
             return {...state,colorPalletVisible: !state.colorPalletVisible}
```
Comment (shubhamkumar648) [inline note-app/src/Reducer/setnoteReducer.js:14]: i like the way you used toggle functionality

**Review by shubhamkumar648 — APPROVED** [review]
good use of functionality and code is readable good to merge

File: note-app/src/Component/Main/Main.jsx [hunk 2]
```diff
@@ -92,7 +104,22 @@ const Main = () => {
...
+              className='tagCont f-s font-l'
+               onClick={() => dispatch({type: "SET_TAG",payload: tag})}
+            >
+              {tag}
+            </div>
+           ))
+         }
+         </div>
+       )}
+         {tags !== '' && <div className='displayTags mt-s ml-s f-s font-l p-xs'>{tags}</div>}
```
Comment (SanketDhabarde) [inline note-app/src/Component/Main/Main.jsx:122]: You can do directly `{tags && <div></div>}`

**Review by SanketDhabarde — APPROVED** [review]
Looks good to me
- Like that, you are being consistent with colors
- The UI looks good

File: note-app/src/Component/Main/Main.jsx (hunk 2 above)
Comment (aniketsingh13) [reply note-app/src/Component/Main/Main.jsx:122]: it show the background of tags in the ui if it is empty. so i use that way

**Pull request merged**